src/components/approval-tasks/ApprovalTasksKebab.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import { ApprovalTaskKind, PipelineRunKind } from '../../types';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+  Tooltip,
+} from '@patternfly/react-core';
+import { KEBAB_BUTTON_ID } from '../../consts';
+import { useTranslation } from 'react-i18next';
+import {
+  useAccessReview,
+  useModal,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { ApprovalTaskModel } from '../../models';
+import { approvalModal } from './modal';
+import { ApproverStatusResponse } from '../../types';
+
+type ApprovalTasksKebabProps = {
+  obj: ApprovalTaskKind;
+  pipelineRun: PipelineRunKind;
+  username: string;
```

Comment:
You can also call `useUser` in the Kabeb component. Btw, I would recommend to call it `ApprovalTaskActionDropdown`. Wdyt?